File: VB.deleteme/Bar.vb
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Namespace HCM401kData.Web
	Public Class Bar
		Public Property Open() As Double
			Get
				Return m_Open
			End Get
			Set
				m_Open = Value
			End Set
		End Property
		Private m_Open As Double
		Public Property High() As Double
			Get
				Return m_High
			End Get
			Set
				m_High = Value
			End Set
		End Property
		Private m_High As Double
		Public Property Low() As Double
			Get
				Return m_Low
			End Get
			Set
				m_Low = Value
			End Set
		End Property
		Private m_Low As Double
		Public Property Close() As Double
			Get
				Return m_Close
			End Get
			Set
				m_Close = Value
			End Set
		End Property
		Private m_Close As Double
		Public Property Volume() As Double
			Get
				Return m_Volume
			End Get
			Set
				m_Volume = Value
			End Set
		End Property
		Private m_Volume As Double
		Public Property TimeStamp() As DateTime
			Get
				Return m_TimeStamp
			End Get
			Set
				m_TimeStamp = Value
			End Set
		End Property
		Private m_TimeStamp As DateTime
	End Class

	Public Class BarInfo
		Inherits List(Of Bar)
		Public Property Token() As HistoryRequest
			Get
				Return m_Token
			End Get
			Set
				m_Token = Value
			End Set
		End Property
		Private m_Token As HistoryRequest
	End Class
End Namespace

File: HCM401kOptimizer/HCM401kAlerter/TA/Bar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace HCM401kAlerter
{
    public class Bar
    {
        public double Open { get; set; }
        public double High { get; set; }
        public double Low { get; set; }
        public double Close { get; set; }
        public double Volume { get; set; }
        public DateTime TimeStamp { get; set; }
    }

    public class BarInfo : List<Bar>
    {
      public  HistoryRequest Token { get; set; }
    }
}
